platform-sdk/swirlds-common/src/main/java/com/swirlds/common/crypto/ImmutableHash.java

```diff
@@ -53,7 +53,6 @@ public ImmutableHash(final Hash mutable) {
      */
     @Override
     public byte[] getValue() {
-        final byte[] value = super.getValue();
-        return Arrays.copyOf(value, value.length);
+        return copyToByteArray();
```

Comment:
Not sure this copy is necessary.  Given the two constructors above, and the rest of the constructors at the parent class `Hash`, I think _all_ constructors do the copy.  ???

Reply:
I did not change the logic, just call a different method. Once `Hash` is immutable, there is no reason for this class to exist.